velox/exec/GroupingSet.cpp

```diff
@@ -714,6 +714,85 @@ void GroupingSet::updateRow(SpillMergeStream& input, char* FOLLY_NONNULL row) {
   }
   mergeSelection_.setValid(input.currentIndex(), false);
 }
+void GroupingSet::abandonPartialAggregation() {
+  abandonedPartialAggregation_ = true;
+  allHaveToIntermediate_ = true;
```

Comment:
nit: some may parse the "HaveTo" as "all must intermediate_". perhaps change to "allSupportToIntermediate".